.github/workflows/ci_web.yml

```
@@ -38,3 +47,8 @@ jobs:
           name: reearth-cms-web
           path: web/reearth-cms-web.tar.gz
           if-no-files-found: error
+        if: failure()
```

actionlint (1.7.4)

key "if" is duplicated in element of "steps" section. previously defined at line:45,col:9

(syntax-check)